docker/base/Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG cert_image
-ARG root_image
+ARG cert_image=alpine:3.19.0
+ARG root_image=alpine:3.19.0
```

Comment:
dependabot doesn't seem to be able to handle this, it did not raise a PR to upgrade to 3.19.1

I think we need to remove these args altogether and define versions directly in the `FROM` directive. That dependabot should definitely support.